addons/images/iss_health_check/README.md

```diff
@@ -0,0 +1,41 @@
+# ISS Health Checker

```

Comment:
We should write out what `ISS` stands for somewhere in this document.

Reply:
Addressed